src/Core/AdminConsole/OrganizationFeatures/OrganizationDomains/VerifyOrganizationDomainCommand.cs

```diff
@@ -112,4 +126,14 @@ private async Task<OrganizationDomain> VerifyOrganizationDomainAsync(Organizatio
 
         return domain;
     }
+
+    private async Task EnableSingleOrganizationPolicyAsync(Guid organizationId)
```

Comment:
Please update https://github.com/bitwarden/server/blob/main/test/Core.Test/AdminConsole/OrganizationFeatures/OrganizationDomains/VerifyOrganizationDomainCommandTests.cs and add a test to check that this happens if the feature flag is enabled